config/development.json

```diff
@@ -133,6 +133,7 @@
 		"signup/design-picker-premium-themes-checkout": true,
 		"signup/hero-flow-non-en": true,
 		"signup/starting-point-courses": true,
+		"signup/redesigned-difm-flow": false,
```

Comment:
Let's enable this because we two are the only one's currently developing on this feature and if somebody wants to see the old screens when in development mode we can guide them to disable this flag locally :)